solutions/src/test/java/ru/sberbank/school/task07/DifferentWordsImplTest.java

```diff
@@ -0,0 +1,43 @@
+package ru.sberbank.school.task07;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class DifferentWordsImplTest {
+
+    private final FileParser fileParser = new FileParserImpl();
+
+    private final DifferentWordsImpl differentWords = new DifferentWordsImpl(fileParser);
+    @Ignore("Сам тест работает, проблема в билде гредла, который не видит русскую кодировку UTF-8")
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1711921819>*